Casks/n/nrfutil.rb

```diff
@@ -0,0 +1,16 @@
+cask "nrfutil" do
+  version "1.0.0-60fe18a"
+  sha256 "229af1183c16cef8e18554ef3a922cb56f16eb7ee8dce76d16a0ace94e7fc6d1"
+
+  url "https://developer.nordicsemi.com/.pc-tools/nrfutil/universal-osx/nrfutil-launcher-universal-apple-darwin-#{version}"
+  name "nrfutil"
+  desc "Unified CLI utility for Nordic Semiconductor products"
+  homepage "https://www.nordicsemi.com/Products/Development-tools/nrf-util"
+
+  livecheck do
+    url "https://developer.nordicsemi.com/.pc-tools/nrfutil/universal-osx/"
+    regex(/nrfutil-launcher-universal-apple-darwin-([\d.]+-[a-z0-9]+)/)
+  end
+
+  binary "nrfutil"
+end
```

Comment:
We can mark this with the below comment as there is no required `zap`

```suggestion
  # No zap stanza required
end
```